ScoutSuite/providers/gcp/services/cloudsql.py

```diff
@@ -82,3 +76,9 @@ def _get_instance_backups(self, instance, params):
         except Exception as e:
             printError('Failed to parse backups for SQL instance %s: %s' % (instance['name'], e))
             return None
+
+    def _is_log_enabled(self, instance) :
+        return instance['settings']['backupConfiguration'].get('binaryLogEnabled')
```

Comment:
If `instance['settings']['backupConfiguration']` doesn't contain `'binaryLogEnabled'`, then `None` will be returned instead of `False`.

```suggestion
        return instance['settings']['backupConfiguration'].get('binaryLogEnabled', False)
```

Reply:
According to the [api reference](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances), `binaryLogEnabled` should be defined. Do we want to assume it is disabled if it is undefined? This solution will appear as "Unknown" in the report. I'm not sure whether it will get flagged or not, I'll investigate.

Reply:
Can confirm, gcp will return False if the config is disabled.

Reply:
As you seem fit, I simply noticed the behaviour changed since your previous implementation of this method!

Reply:
Alright thanks!